tracer/contrib/sqltraced/sqlutils/testing.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
```

Comment:
Hum actually this package needs to be exported since its function `AllSQLTests` is also used to test sqlxtraced package. Maybe should I rename it sqltracedtest to be more clear?